katas/joaquin-gioffre/main/Week-01/killer-garage-door/Door.cs

```diff
@@ -1,4 +1,4 @@
-namespace KillerGarageDoor;
```

Comment:
Since you are trying to following best coding practices, it is a good idea to use enums and constants when possible, e.g. State could be an enum like DoorState.Closing and DoorState.Opening

Also, Position could be evaluated against a constant OPENED_POSITION = 5 or CLOSED_POSITION = 0

Reply:
I added the constants to be evaluated against the Position but I'm not sure how I could implement the enums for the state in this case. Thanks for the observation because I'm not so used to implement enums and constants so this helps me to start thinking in them when I code 🙌🏻